libretroshare/src/retroshare/rspeers.h

```diff
@@ -599,6 +627,24 @@ class RsPeers
 	        bool includeSignatures = false,
 	        bool includeExtraLocators = true ) = 0;
 
+	/**
+	 * @brief Get RetroShare short invite of the given peer
+	 * @jsonapi{development}
+	 * @param[out] invite storage for the generated invite
+	 * @param[in] sslId Id of the peer of which we want to generate an invite,
+	 *	a null id (all 0) is passed, an invite for own node is returned.
+	 * @param[in] formatRadix true to get in base64 format false to get URL.
+	 * @param[in] includeFingerprint true to include PGP fingerprint in the URL.
+	 * @param[in] bareBones true to get smalles invite, which miss also
+	 *	the information necessary to attempt an outgoing connection, but still
+	 *	enough to accept an incoming one.
+	 * @return false if error occurred, true otherwise
+	 */
+	virtual bool getShortInvite(
+	        std::string& invite, const RsPeerId& sslId = RsPeerId(),
+	        bool formatRadix = false, bool includeFingerprint = false,
```

Comment:
Same here. Why have options in the short invite? The format should be decided and enforced.

Reply:
It is convenient to put more or less information/fields depending on what is the medium of transmission, we have someway similar mechanism with in long invites. About the format we already do something similar with long invites too, depending on the medium of transmission it might be more convenient to have it in URI format or in radix64 format. This implement it at libretroshare level instead of delegating the implementation to the GUI with consequent problems in keeping compatibility between different GUI (when I programmed the Qml app I had an hard time to keep RS links compatibles between RS GUI and Qml App because of this)